lib/models/dish.rb

```diff
@@ -0,0 +1,7 @@
+class Dish
+  include DataMapper::Resource
+
+  property :id, Serial, key: true
+  property :dish_name, String, length: 128
```

Comment:
Instead of `dish_name`, consider using just `name`. This attribute name you choose will spread through your entire program - you can see it already in your tests - that you are putting `dish_name` instead of `name` in the `given the following dishes exist`.